modules/cluster-cloud-controller-manager-operator.adoc

```diff
@@ -10,9 +10,9 @@
 
 [NOTE]
 ====
-This Operator is General Availability for Microsoft Azure Stack Hub, Nutanix, {rh-openstack-first}, and VMware vSphere.
+This Operator is General Availability for Amazon Web Services (AWS), global Microsoft Azure, Microsoft Azure Stack Hub, Nutanix, {rh-openstack-first}, and VMware vSphere.
 
-It is available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] for Alibaba Cloud, Amazon Web Services (AWS), Google Cloud Platform (GCP), IBM Cloud, IBM Cloud Power VS, and Microsoft Azure.
+It is available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] for Alibaba Cloud, Google Cloud Platform (GCP), IBM Cloud, and IBM Cloud Power VS.
```

Comment:
Consider switching the serial list at the end of the sentence to an unordered list. Better scalability.